draft-private-access-tokens.md

```diff
@@ -1084,11 +1049,173 @@ enc, context = SetupBaseR(enc, skI, "AccessTokenRequest")
 origin_name, error = context.Open(aad, ct)
 ~~~
 
+## Stable Mapping Computation {#stable-mapping}
+
+This section describes the Client, Mediator, and Issuer behavior in computing
+the stable mapping based on client identity and origin name. At a high level,
+this functionality computes y = F(x, k), where x is a per-Client secret and
+k is a per-Origin secret, subject to the following constraints:
+
+- The Mediator only learns y if the Client in possession of x engages with the protocol;
+- The Client cannot engage in the protocol for private input x' that is not equal to x; and
+- The Issuer does not learn x, nor does it learn when two requests correspond to the same private value x.
```

Comment:
The wording here is a little confusing, maybe reword cannot engage with mediator prevents? Also say something about what people learn about k?